lib/runCall.js

```diff
@@ -28,7 +29,7 @@ module.exports = function (opts, cb) {
   var toAccount
   var toAddress = opts.to
   var createdAddress
-  var txValue = opts.value || new Buffer(0)
+  var txValue = opts.value || Buffer.alloc(0)
```

Comment:
You see, this `alloc` feature is just confusing 😉 

I think in this case though both are acceptable as value can be zero or empty (still meaning zero).

Eventually all the values should be kept as BN and not as a Buffer.